versioned_docs/version-0.5.0/release-notes/release-notes-for-kubeslice-oss-0.5.1.mdx

```diff
@@ -0,0 +1,50 @@
+# Release Notes for KubeSlice OSS 0.5.1
+
+*Release date: 06 Jan 2023*
+
+KubeSlice is a ***cloud-independent*** platform that combines network, application, Kubernetes, and deployment services 
+in a framework to accelerate application deployment in a multi-cluster and multi-tenant environment. KubeSlice achieves this 
+by creating logical application ***slice*** boundaries that enable seamless communication between pods and services across 
+clusters, clouds, edges, and data centers.
+
+We continue to add new features and enhancements to KubeSlice. 
+
+## What's New
+These release notes describe the new changes and enhancements in this version.
+
+### Network Service Mesh Upgrade 
+The Network Service Mesh (NSM) component has been upgraded to the stable 
+[GA version 1.5.0](https://networkservicemesh.io/docs/releases/v1.5.0/) that provides upstream networking fixes. 
+
+### Latest Supported Kubernetes Version 
+The latest Kubernetes version that we support from this release is version [1.24](https://v1-24.docs.kubernetes.io/). 
+
+### Enhancements 
+* When the **namespace sameness** is applied to a namespace on a slice, then it applies to all worker 
+  clusters that are part of the slice. If a worker cluster does not already have that namespace, it is now created. This 
+  ensures that all the worker clusters that are part of a slice will have that 
+  namespace for which the namespace sameness is applied. This created namespace remains on the worker 
+  cluster even after the worker cluster is detached from that slice, and even when that slice is deleted.
+* The **kubeslice-cli** tool now allows you to uninstall Kubeslice components using a custom topology file.
```

Comment:
Only this info should be there 

The kubeslice-cli tool now allows uninstalling KubeSlice components ...

Reply:
updated